use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -162,15 +176,18 @@ main(int argc, char *argv[])
     exit(EXIT_SUCCESS);
 #endif
 
-    clock_id = CLOCK_THREAD_CPUTIME_ID;
+    // Default, so throughput (nops/sec) is consistent with normal expectation.
+    clock_id = CLOCK_REALTIME;
 
     // Arg-parsing is only supported on Linux.
     int rv = parse_arguments(argc, argv, &clock_id);
     if (rv) {
         errExit("Argument error.");
     }
 
-    svr_clock_getres();
+#if !defined(L3_ENABLED)
```

Comment:
Is this right? Why only call `svr_clock_calibrate()` if there is no logging enabled? It looks like this artificially reduces the reported performance when logging is enabled.

Reply:
Think of this as a diagnostic code now. I added this to understand what the impact of a call to `clock_gettime()` is.

Previously, I played around with ways to deduct this overhead from the overall elapsed time computation, to see if it makes a difference. It did not, so I don't really use the calibrated overhead value anywhere in metrics computation.

Also, note this call occurs on L188 whereas the actual `start` begins much later in the code:

```
260     if (clock_gettime(clock_id, &ts0)) {    // ***** Timing begins
261         errExit("clock_gettime-ts0");
262     }
```

So, your comment below is also moot and this `calibrate` call is a red herring.